src/raster/r.futures/r.futures.devpressure/r.futures.devpressure.html

```diff
@@ -85,4 +85,4 @@ <h2>AUTHOR</h2>
 
 Anna Petrasova, <a href="https://geospatial.ncsu.edu/geoforall/">NCSU GeoForAll</a><br>
 
-<p><i>Last changed: $Date: 2018-09-14 21:49:22 -0400 (Fri, 14 Sep 2018) $</i>
+<!-- <p><i>Last changed: $Date: 2018-09-14 21:49:22 -0400 (Fri, 14 Sep 2018) $</i> -->
```

Comment:
I have a style question for @neteler, concerning my understanding of how these dates were abandoned in the link you have pointed me on another PR https://trac.osgeo.org/grass/wiki/Submitting/Docs#Footer .
Since it is not useful anymore, and when I change the file the existing "Last changed" date is now invalid, it is the good way to go to comment it, or should I remove them of the code? Commented out is pointless, except for maybe git blame that is cleaner to find? Is the commented HTML stripped out of the generated web page? If not, it might be useful to remove when applicable to prevent any potential parsing oddities, or for search engines if ever they index this.

Reply:
Well, we still hope to invent a way to reactivate these file dates with git. That's why they are still there.
Suggestions are welcome.

In any case, there must be no date but just the variable. I have fixed above line in #667

Reply:
Can I leave the commit changes as-it for this PR, or I need to revert to avoid merge conflicts? One of the two PRs will need to handle a merge conflict anyways.

Reply:
Since this is larger, I'll take care of the merge conflicts in my smaller PR.